analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextBaseTest.ShouldAnalyzeTree.cs

```diff
@@ -78,18 +77,17 @@ public void ShouldAnalyzeTree_Scanner_UnchangedFiles_ContainsOtherFile()
     [DataRow(OtherFileName, true)]
     public void ShouldAnalyzeTree_GeneratedFile_NoSonarLintXml(string fileName, bool expected)
     {
-        var sonarLintXml = CreateSonarLintXml(true);
+        var sonarLintXml = new DummySourceText(AnalysisScaffolding.GenerateSonarLintXmlContent(analyzeGeneratedCode: true));
         var (compilation, tree) = CreateDummyCompilation(AnalyzerLanguage.CSharp, fileName);
         var sut = CreateSut(compilation, CreateOptions(sonarLintXml, @"ResourceTests\Foo.xml"));
-
```

Comment:
This new line should not be removed. It is there to help read the test. To split the Arrange and Act/Assert steps